core/spec/lib/spree/event_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'spree/event'
+
+RSpec.describe Spree::Event do
+  let(:subscription_name) { 'foo_bar' }
+  let(:item) { spy('object') }
+  let(:notifier) { ActiveSupport::Notifications.notifier }
+
+  subject { described_class }
+
+  it 'has default adapter' do
+    expect(subject.adapter).to eql Spree::Event::Adapters::ActiveSupportNotifications
+  end
+
+  before do
+    # ActiveSupport::Notifications expose no interface to clean all
```

Comment:
I'd word this differently, like `ActiveSupport::Notifications does not provide an interface...`, WDYT?

Reply:
👍